session-09-lab1-gitops-intro/step1.md

```diff
@@ -27,7 +27,7 @@ To Fork the repository:
 
 1. Make sure you are logged into your Github Account at `https://github.com/firstname-lastname-slalom`
 1. Open `https://github.com/fluxcd/flux-get-started` in your browser
-1. Click the ![Fork Icon](repo-fork.png) icon
+1. Click the ![Fork Icon](assets/repo-fork.png) icon
```

Comment:
You dont actually need this and my experience was the link breaks when you add assets